package.json

```diff
@@ -13,21 +13,22 @@
     "errorhandler": "^1.2.1",
     "event-stream": "^3.1.7",
     "express": "^4.9.7",
-    "frozen-moment": "^0.1.1",
-    "lodash": "^2.4.1",
+    "frozen-moment": "^0.4.0",
+    "kerberos": "~0.0",
+    "lodash": "^4.17.4",
     "ludwig-api": "^1.2.0",
     "moment": "^2.8.3",
-    "mongoose": "~3.8.18",
+    "mongoose": "^4.8.3",
     "morgan": "^1.3.2",
-    "superagent": "^0.21.0",
-    "underscore.string": "^2.3.3"
+    "superagent": "^3.4.4",
+    "underscore.string": "^3.3.4"
```

Comment:
This dependency is only used in a migration (`acceptance-tests-v2`). Should at least be moved to a `devDependency` or  an `optional` dependency (if not removed altogether with a comment added next to the `require` statement explaining which version to install), and not upgraded unless absolutely necessary as this code is never tested nor run.